src/qml/editorwidgets/ValueMap.qml

```diff
@@ -125,7 +125,7 @@ EditorWidgetBase {
         iconSource: Theme.getThemeIcon("ic_baseline_search_black")
         iconColor: Theme.mainTextColor
 
-        visible: enabled
+        visible: enabled && comboBox.count > 6 // Make value map searchable only if it contains more than 6 items
```

Comment:
How did we decide the number 6? I guess a comment not only what, but also why we have this numbre in the code would make it easy for future us.

We can also make it a little bit less magic number and set it as a const somewhere where we can explain the whole background behind this PR?

Reply:
The search button introduced in #5192 eat up quite some space. For value maps with less than X elements it makes not really sense to make them searchable. With some tries I found out that with X=6 it was easy to find an element and they where all visible at once with some reserve on my "normal" sized phone. Of course this is very empirical and not based on any UX design study so I am happy to discuss about that.

Where would you put the const? into `Theme.qml`?
In general there are quite a lot of magic numbers around mostly for spacing, sizes and z values. Should we try to put theme to Theme.qml when possible?

Another question is if the searchable view could substitute the combobox popup entirely? Then there would be no need for the search button and there are no more two different interfaces to choose items